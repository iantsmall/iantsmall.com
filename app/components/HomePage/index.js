/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 */

import React from 'react';
import { Helmet } from 'react-helmet';
import { Container, Row, Col, Jumbotron } from 'react-bootstrap';

import Menu from 'containers/Menu/Loadable';

export default function HomePage() {
  return (
    <React.Fragment>
      <Helmet>
        <title>Hex in React Redux</title>
        <meta name="description" content="Portfolio of Ian T. Small" />
      </Helmet>
      <Container>
        <Row>
          <Col xs={12}>
            <Menu />
          </Col>
        </Row>
        <Row>
          <Col xs={12}>
            <div id="alert" />
          </Col>
        </Row>
        <Row>
          <Col xs={12}>
            <Jumbotron id="viewport" className="modal-container">
              <h1>More Content Coming Soon</h1>
            </Jumbotron>
          </Col>
        </Row>
      </Container>
    </React.Fragment>
  );
}
