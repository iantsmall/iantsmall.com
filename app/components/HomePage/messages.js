/*
 * MenuView Messages
 *
 * This contains all the text for the MenuView component.
 */
// import React from 'react';
import { defineMessages } from 'react-intl';

export const scope = 'app.components.HomePage';

export default defineMessages({
  resumeheader: {
    id: `${scope}.resumeheader`,
    defaultMessage: 'Fullstack Developer Resume',
  },
});
