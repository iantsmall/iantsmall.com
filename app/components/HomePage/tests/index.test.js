import React from 'react';
import { cleanup } from 'react-testing-library';
import render from 'utils/testRenderWithContext';

import HomePage from '../index';

describe('<HomePage />', () => {
  afterEach(cleanup);
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<HomePage />);
    expect(firstChild).toMatchSnapshot();
  });
});
