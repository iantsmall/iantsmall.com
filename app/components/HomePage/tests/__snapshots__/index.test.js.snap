// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render and match the snapshot 1`] = `
<div
  class="container"
>
  <div
    class="row"
  >
    <div
      class="col-12"
    />
  </div>
  <div
    class="row"
  >
    <div
      class="col-12"
    >
      <div
        id="alert"
      />
    </div>
  </div>
  <div
    class="row"
  >
    <div
      class="col-12"
    >
      <div
        class="modal-container jumbotron"
        id="viewport"
      >
        <h1>
          More Content Coming Soon
        </h1>
      </div>
    </div>
  </div>
</div>
`;
